x-pack/plugins/cases/server/services/cases/transform.ts

```diff
@@ -214,7 +214,19 @@ export function transformSavedObjectToExternalModel(
   return {
     ...caseSavedObject,
     attributes: {
-      ...caseSavedObjectAttributes,
+      description: caseSavedObjectAttributes.description,
+      title: caseSavedObjectAttributes.title,
+      tags: caseSavedObjectAttributes.tags,
+      settings: caseSavedObjectAttributes.settings,
+      owner: caseSavedObjectAttributes.owner,
+      assignees: caseSavedObjectAttributes.assignees,
+      duration: caseSavedObjectAttributes.duration,
+      closed_at: caseSavedObjectAttributes.closed_at,
+      closed_by: caseSavedObjectAttributes.closed_by,
+      created_at: caseSavedObjectAttributes.created_at,
+      created_by: caseSavedObjectAttributes.created_by,
```

Comment:
What about fields like `closed_by`, `created_by`, `connector`, and `updated_by`? Should we explicitly map them too?